db/migrate/20241129053223_change_status_to_integer_in_todos.rb

```diff
@@ -0,0 +1,35 @@
+class ChangeStatusToIntegerInTodos < ActiveRecord::Migration[7.2]
```

Comment:
for now its fine but this is not a good practice, it will put a huge load on DB in case of large data set

Reply:
okay